src/media/js/components/handlers/addons/submit.js

```diff
@@ -3,10 +3,10 @@ import {bindActionCreators} from 'redux';
 import FileReaderInput from 'react-file-reader-input';
 import {connect} from 'react-redux';
 
-import {validate as validateAddon} from '../../actions/submissionAddon';
+import {validate as validateAddon} from '../../../actions/submissionAddon';
```

Comment:
Maybe we should consolidate files by content type so that related files aren't so far away from one another. (E.g, js/addon, js/langpack) like Django apps.